PyDSS/valiate_settings.py

```diff
@@ -0,0 +1,130 @@
+import os
+
+settings_dict = {
+        "Exports": {
+            'Export Mode': {'type': str, 'Options': ["byClass", "byElement"]},
+            'Export Style': {'type': str, 'Options': ["Single file", "Separate files"]},
+            # Feather is not supported because its underlying libraries do not support complex numbers
+            'Export Format': {'type': str, 'Options': ["csv", "h5"]},
+            'Export Compression': {'type': bool, 'Options': [True, False]},
+            'Export Iteration Order': {'type': str, 'Options': ["ElementValuesPerProperty",
+                                                                "ValuesByPropertyAcrossElements"]},
+            'Export Elements': {'type': bool, 'Options': [True, False]},
+            'Export Event Log': {'type': bool, 'Options': [True, False]},
+            'Export Data Tables': {'type': bool, 'Options': [True, False]},
+            'Export Data In Memory': {'type': bool, 'Options': [True, False]},
+            'HDF Max Chunk Bytes': {'type': int, 'Options': range(16 * 1024, 1024 * 1024 + 1)},
+            'Log Results': {'type': bool, 'Options': [True, False]},
+            'Result Container': {'type': str, 'Options': ['ResultContainer', 'ResultData']},
+        },
+        "Frequency": {
+            'Enable frequency sweep': {'type': bool, 'Options': [True, False]},
+            'Fundamental frequency': {'type': int, 'Options': [50, 60]},
+            'Start frequency': {'type': float},
+            'End frequency': {'type': float},
+            'frequency increment': {'type': float},
+            'Neglect shunt admittance': {'type': bool, 'Options': [True, False]},
+            'Percentage load in series': {'type': float, 'Options': range(0, 100)},
+        },
+        "Helics": {
+            'Co-simulation UUID': {'type': str},
```

Comment:
This is not defined in `defaults/simulation.toml`, so if you don't add it to the project simulation.toml file you get an error. It might make sense for this to be a required parameter if you are using HELICS. It is not a good user experience to make this required if you are not using HELICS.

Reply:
This also resulted from wrong merge. Issue has been fixed